gradle/deploySnapshot.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: "com.jfrog.artifactory"
```

Comment:
Could you switch this to single quotes to be consistent with the other build scripts?